code/modules/client/preferences/_preferences.dm

```diff
@@ -1219,13 +1221,35 @@ GLOBAL_LIST_INIT(name_adjustments, list())
 						skin_tone = listy[new_s_tone]
 
 				if("selected_accent")
-					if(!patreon)
-						to_chat(user, "Sorry this is a patreon exclusive feature.")
+					if(istype(pref_species, /datum/species/human/halfelf) || istype(pref_species, /datum/species/human/halfdrow))
```

Comment:
```suggestion
					if(istype(pref_species, /datum/species/human/halfelf) || istype(pref_species, /datum/species/human/halfdrow))
```
can you tie this with the species instead of hardcoding it?